src/EbmlDummy.cpp

```diff
@@ -10,7 +10,9 @@
 
 namespace libebml {
 
-DEFINE_EBML_CLASS_ORPHAN(EbmlDummy, 0xFF, 1, "DummyElement", EbmlDocVersion{} )
+static constexpr EbmlDocVersion AllEbmlVersion{};
```

Comment:
In the tests you use the plural, and using the plural here looks to be more correct to me. Please change to `AllEbmlVersions`. Thanks.

Reply:
Done. Also I think it would be good to move all these element definitions in a single file, similar to how it's done in libmatroska. But I'll keep that for later.